pom.xml

```diff
@@ -38,20 +38,26 @@
 		<dependency>
 			<groupId>commons-lang</groupId>
 			<artifactId>commons-lang</artifactId>
-			<version>2.1</version>
+			<version>2.6</version>
```

Comment:
Had to be updated due to the new parent pom's ["upper-bounds-deps" maven-enforcer-plugin rule](https://maven.apache.org/components/enforcer/enforcer-rules/requireUpperBoundDeps.html).